docs/src/components/Tabs.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import PropType from 'prop-types';
+
+export const TabItem = (props) => {
+	return props.children;
+};
+TabItem.propTypes = {
+	children: PropType.oneOfType([PropType.string, PropType.array]),
+};
+
+export const Tabs = (props) => {
+	const data = props.children || [];
+	const [visibleTab, setVisibleTab] = useState(0);
+	const listTitles = data.map((item, index) => (
+		<li key={`tab${index}`}>
```

Comment:
Because you're going to need to pass in an id, use that for a key instead of index.  

Avoid whenever possible as using index as a key is an anti-pattern for react.